Update audit-logs-troubleshooting.md with Loki example (#136)
giantswarm/handbook: content/docs/support-and-ops/ops-recipes/audit-logs-troubleshooting.md

```bash
cat /tmp/audit.log | jq '. | select(.objectRef.name=="prometheus-prometheus-exporters-tls-assets")'
```

## Using Loki

> Now all installations >= 19.1.0 have Loki instance deployed with audit logs included. So we can leverage on Lok to get the logs. Example of query:

Reviewer: All AWS vintage installations have loki deployed,
but WCs logs are not collected yet.

When they will be collected, that will be for WCs >= 19.1.0, but we have some more checks to perform before we turn WC logs collection on.

Reviewer 2: > Do we have a expected date for that? what about onprem, is Loki deployed there too? or planning?

I asked myself the same question, when I saw your original audit log documentation page and the Loki audit feature announcement on Slack (when to look where).

Reviewer: For the moment we're focusing on Loki working well on AWS vintage, then we will add more cloudproviders.

WCs collection is under test on a few `anteater` clusters. We have some tuning to do on Loki for it to handle the load, and some cost estimation.

Onprem will probably be the most difficult one to add, because Loki relies on object storage and there's no standard object storage for capvcd.

Author: Ok I am adding a note with the supported envs

> Now all installations >= 19.1.0 have Loki instance deployed with audit logs included. So we can leverage on Lok to get the logs. Example of query:

Author: ```suggestion
Today all vintage Management Clusters have Loki instance deployed with audit logs included. So we can leverage on Loki to get the logs. Future weeks we will extend Loki to vintage Workload Clusters too. Example of query:
```

> Today all vintage Management Clusters have Loki instance deployed with audit logs included. So we can leverage on Loki to get the logs. Future weeks we will extend Loki to vintage Workload Clusters too. Example of query:

Reviewer 2: ```suggestion
Today all vintage Management Clusters have a Loki instance deployed with audit logs included. So we can leverage Loki to get the logs. We will also roll out Loki on vintage Workload Clusters in the coming weeks. Example of query:
```

```

> {cluster_id="myCluster",scrape_job="audit-logs"} |= `` | json | user_username=`joe@giantswarm.io`

Reviewer 2: ```suggestion
{cluster_id="myCluster",scrape_job="audit-logs"} |= `` | json | user_username=`johndoe@example.com`
```

```

> __Note__: In json filter to access nested properties you use `_` for getting a child property as the example above (user.username -> user_username).

Reviewer: nice one :+1:

> __Note__: In json filter to access nested properties you use `_` for getting a child property as the example above (user.username -> user_username).

Reviewer 2: ```suggestion
__Note__: Use `_` in json filters to access properties. In the example query above `user.username` is specified as `user_username`.
```
